app/src/main/java/com/namdinh/cleanarchitecture/presentation/main/CheckStyleTest.java

```diff
@@ -0,0 +1,21 @@
+package com.namdinh.cleanarchitecture.presentation.main;
+
+
+import com.namdinh.cleanarchitecture.presentation.base.activity.BaseActivity;
+
+import android.os.storage.OnObbStateChangeListener;
+import android.os.storage.OnObbStateChangeListener;
+import android.os.storage.OnObbStateChangeListener;
```

Comment:
ERROR: Duplicate import to line 6 - android.os.storage.OnObbStateChangeListener.